components/_buttons.scss

```diff
@@ -79,6 +82,10 @@ button, input[type="submit"] {
     @include button-color($tws-yellow);
   }
 
+  &.fb-blue {
+    @include button-color(#435F9D);
```

Comment:
Color literals like `#435F9D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#435F9D` should be written as `#435f9d`